src/Symfony/Component/HttpFoundation/Tests/Session/Storage/NativeSessionStorageTest.php

```diff
@@ -272,4 +272,28 @@ public function testRestart()
         $this->assertSame($id, $storage->getId(), 'Same session ID after restarting');
         $this->assertSame(7, $storage->getBag('attributes')->get('lucky'), 'Data still available');
     }
+
+    public function testCanCreateNativeSessionStorageWhenSessionAlreadyStarted()
+    {
+        session_start();
+        $this->getStorage();
+
+        // Assert no exception has been thrown by `getStorage()`
+        $this->assertTrue(true);
+    }
+
+    /**
+     * @expectedException \LogicException
+     */
+    public function testCannotSetSessionOptionsOnceSessionStarted()
+    {
+        if (PHP_VERSION_ID < 70200) {
```

Comment:
Annotation `@requires PHP 7.2.0`

Reply:
Would that allow 7.2 and above or lock at 7.2.* or 7.(2+) ?

Reply:
`@requires PHP 7.2` works